potato/flask_server.py

```diff
@@ -93,7 +96,18 @@
     "rgb(179, 179, 179)",
 ]
 
-app = Flask(__name__)
+
+
+args = arguments()
+init_config(args)
+if config.get("verbose"):
+    logger.setLevel(logging.DEBUG)
+if config.get("very_verbose"):
+    logger.setLevel(logging.NOTSET)
+
+
+app = create_app(os.path.join(POTATO_HOME, config["db_path"]))

```

Comment:
Unfortunately bringing this out of `main()` again because of the dependency between `config` and `create_app()`, and the fact that `app` needs to be out here for the routes to work (ie `@app.route("/")`).